web/src/components/FacetInput.tsx

```diff
@@ -54,7 +56,7 @@ const FacetInput: React.FC<FacetInputProps> = ({
       name={name}
       value={value}
       placeholder={placeholder}
-      type='number'
+      type={type || 'number'}
```

Comment:
i would suggest to set the `type` as required and pass it also for the "number" input field. Then you don't need the `||` operator